server/container/guice/mailbox-plugin-deleted-messages-vault-guice/src/main/java/org/apache/james/modules/vault/DeletedMessageVaultModule.java

```diff
@@ -42,29 +40,16 @@
 
 public class DeletedMessageVaultModule extends AbstractModule {
```

Comment:
I would have had introduced it only as a separate module that we could have been using just for overrides: here things will break!